app.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const { getTopics } = require('./controllers/app.controllers');
+const { handleRouteNotFoundError } = require('./controllers/errors.controllers');
+const app = express();
+
+app.use(express.json());
```

Comment:
As we're not yet making a post request, this line of code shouldn't currently be included.
It's important to only include code that meets the need of the current ticket/task

Reply:
deleted it!